cuda_bindings/cuda/bindings/_internal/nvvm_windows.pyx

```diff
@@ -0,0 +1,373 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+#
+# This code was automatically generated across versions from 12.0.1 to 12.8.0. Do not modify it directly.
+
+from libc.stdint cimport intptr_t
+
+from .utils cimport get_nvvm_dso_version_suffix
+
+from .utils import FunctionNotFoundError, NotSupportedError
+
+import os
+import site
+
+import win32api
+
+
+###############################################################################
+# Wrapper init
+###############################################################################
+
+LOAD_LIBRARY_SEARCH_SYSTEM32     = 0x00000800
+LOAD_LIBRARY_SEARCH_DEFAULT_DIRS = 0x00001000
+LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR = 0x00000100
+cdef bint __py_nvvm_init = False
+cdef void* __cuDriverGetVersion = NULL
+
+cdef void* __nvvmVersion = NULL
+cdef void* __nvvmIRVersion = NULL
+cdef void* __nvvmCreateProgram = NULL
+cdef void* __nvvmDestroyProgram = NULL
+cdef void* __nvvmAddModuleToProgram = NULL
+cdef void* __nvvmLazyAddModuleToProgram = NULL
+cdef void* __nvvmCompileProgram = NULL
+cdef void* __nvvmVerifyProgram = NULL
+cdef void* __nvvmGetCompiledResultSize = NULL
+cdef void* __nvvmGetCompiledResult = NULL
+cdef void* __nvvmGetProgramLogSize = NULL
+cdef void* __nvvmGetProgramLog = NULL
+
+
+cdef inline list get_site_packages():
+    return [site.getusersitepackages()] + site.getsitepackages()
+
+
+cdef load_library(const int driver_ver):
+    handle = 0
+
+    for suffix in get_nvvm_dso_version_suffix(driver_ver):
+        if len(suffix) == 0:
+            continue
+        dll_name = f"nvvm_{suffix}0_0.dll"
```

Comment:
Please check: I think for all CUDA 12.x this is
```suggestion
        dll_name = f"nvvm64_40_0.dll"
```
xref: https://github.com/NVIDIA/nvmath-python/blob/073b168ac0688fa3b84caaa8bb65948bf7db7eae/nvmath/_utils.py#L155-L200